bridge_adaptivity/module/urls.py

```diff
@@ -4,14 +4,24 @@
 
 from module.views import (
     ActivityCreate, ActivityDelete, ActivityUpdate, callback_sequence_item_grade, CollectionCreate, CollectionDetail,
-    CollectionList, CollectionUpdate, sequence_item_next, SequenceComplete, SequenceItemDetail
+    CollectionList, CollectionUpdate, GroupCreate, GroupDetail, GroupList, GroupUpdate,
+    sequence_item_next, SequenceComplete, SequenceItemDetail
 )
 
 urlpatterns = [
-    url(r'^collection/$', CollectionList.as_view(), name='collection-list'),
-    url(r'^collection/add/$', CollectionCreate.as_view(), name='collection-add'),
-    url(r'^collection/(?P<pk>\d+)/change/$', CollectionUpdate.as_view(), name='collection-change'),
-    url(r'^collection/(?P<pk>\d+)/$', CollectionDetail.as_view(), name='collection-detail'),
+    url(r'^group/$', GroupList.as_view(), name='group-list'),
+    url(r'^group/add/?$', GroupCreate.as_view(), name='group-add'),
+    url(r'^group/(?P<pk>\d+)/$', GroupDetail.as_view(), name='group-detail'),
+    url(r'^group/(?P<pk>\d+)/change/$', GroupUpdate.as_view(), name='group-change'),
+
+    url(r'^(?:group/(?P<group_slug>\w+)/)?collection/$', CollectionList.as_view(), name='collection-list'),
```

Comment:
L17-23 you use the two same tools please choose one of them `(?:group/(?P<group_slug>\w*)/)` or `(group/(?P<group_slug>\w+)/)?`